Assets/Scripts/HPPickupInteract.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+public class HPPickupInteract : MonoBehaviour
+{
+    // Private Variables \\
+    [SerializeField]
+    private int hpPickupVal { get; set; } = 100;
```

Comment:
100 seems pretty high to me because it will always heal the player fully, but we can adjust it later.

Reply:
Yeah in my mind 100 was just a good "default" value since I expected the actual value to be multiple different numbers dependent on things implemented in the future. If you have a number in mind I wouldn't mind setting it to that though.